django_prices_openexchangerates/tests.py

```diff
@@ -28,47 +28,27 @@ def mock_rates(currency):
             side_effect=mock_rates)
 class CurrencyConversionTestCase(TestCase):
     def test_the_same_currency_uses_no_conversion(self, mock_qs):
-        price = Price(10, currency='USD')
+        price = Amount(10, currency='USD')
         converted = exchange_currency(price, 'USD')
-        self.assertIsNone(converted.history)
         self.assertEqual(converted, price)
 
     def test_base_currency_to_another(self, mock_qs):
-        converted = exchange_currency(Price(10, currency='USD'), 'EUR')
+        converted = exchange_currency(Amount(10, currency='USD'), 'EUR')
         self.assertEqual(converted.currency, 'EUR')
-        self.assertIsNotNone(converted.history)
-        modifier = converted.history.right
-        self.assertEqual(modifier.base_currency, 'USD')
-        self.assertEqual(modifier.to_currency, 'EUR')
+        self.assertIsNotNone(converted)
 
     def test_convert_another_to_base_currency(self, mock_qs):
-        base_price = Price(10, currency='EUR')
+        base_price = Amount(10, currency='EUR')
         converted_price = exchange_currency(base_price, 'USD')
         self.assertEqual(converted_price.currency, 'USD')
-        self.assertIsNotNone(converted_price.history)
-        # 1 / rate should be used to calculate final amount
-        conversion = converted_price.history.right
-        self.assertEqual(conversion.base_currency, 'EUR')
-        self.assertEqual(conversion.to_currency, 'USD')
-        self.assertEqual(conversion.rate, 1 / RATES['EUR'])
 
     def test_convert_two_non_base_currencies(self, mock_qs):
-        base_price = Price(10, currency='EUR')
+        base_price = Amount(10, currency='EUR')
         converted_price = exchange_currency(base_price, 'GBP')
         self.assertEqual(converted_price.currency, 'GBP')
-        self.assertIsNotNone(converted_price.history)
-        # Price should have two modifiers
-        # EUR to USD and then USD to GBP
-        first_operation = converted_price.history.left.history
-        self.assertEqual(first_operation.left, base_price)
-        self.assertEqual(first_operation.right.base_currency, 'EUR')
-        self.assertEqual(first_operation.right.to_currency, 'USD')
-        second_operation = converted_price.history.right
-        self.assertEqual(second_operation.base_currency, 'USD')
-        self.assertEqual(second_operation.to_currency, 'GBP')
 
     def test_convert_price_uses_passed_dict(self, mock_qs):
-        base_price = Price(10, currency='USD')
+        base_price = Amount(10, currency='USD')
```

Comment:
This name is misleading as it's not a Price instance.

Reply:
Yes, i will update tests shortly.